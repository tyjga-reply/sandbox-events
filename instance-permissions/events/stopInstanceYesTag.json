{
    "version": "0",
    "id": "34a91063-1d04-e7e5-593a-b7e4bad51f09",
    "detail-type": "AWS API Call via CloudTrail",
    "source": "aws.ec2",
    "account": "211125361846",
    "time": "2024-12-17T12:14:49Z",
    "region": "us-east-1",
    "resources": [],
    "detail": {
        "eventVersion": "1.10",
        "userIdentity": {
            "type": "AssumedRole",
            "principalId": "AROATCKANLS3NIDXYUGTV:t.kapp@reply.de",
            "arn": "arn:aws:sts::211125361846:assumed-role/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4/t.kapp@reply.de",
            "accountId": "211125361846",
            "accessKeyId": "ASIATCKANLS3I7CEFORI",
            "sessionContext": {
                "sessionIssuer": {
                    "type": "Role",
                    "principalId": "AROATCKANLS3NIDXYUGTV",
                    "arn": "arn:aws:iam::211125361846:role/aws-reserved/sso.amazonaws.com/eu-central-1/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4",
                    "accountId": "211125361846",
                    "userName": "AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4"
                },
                "attributes": {
                    "creationDate": "2024-12-17T08:29:50Z",
                    "mfaAuthenticated": "false"
                }
            }
        },
        "eventTime": "2024-12-17T12:14:49Z",
        "eventSource": "ec2.amazonaws.com",
        "eventName": "StopInstances",
        "awsRegion": "us-east-1",
        "sourceIPAddress": "91.102.9.132",
        "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.1.1 Safari/605.1.15",
        "requestParameters": {
            "instancesSet": {
                "items": [
                    {
                        "instanceId": "i-093127c3e6bddd5a6"
                    }
                ]
            },
            "force": false
        },
        "responseElements": {
            "requestId": "e597cfad-4cde-4965-93c5-c9e2c43af1af",
            "instancesSet": {
                "items": [
                    {
                        "instanceId": "i-093127c3e6bddd5a6",
                        "currentState": {
                            "code": 64,
                            "name": "stopping"
                        },
                        "previousState": {
                            "code": 16,
                            "name": "running"
                        }
                    }
                ]
            }
        },
        "requestID": "e597cfad-4cde-4965-93c5-c9e2c43af1af",
        "eventID": "7633b462-f95e-4e8e-b42c-449ddb38c76d",
        "readOnly": false,
        "eventType": "AwsApiCall",
        "managementEvent": true,
        "recipientAccountId": "211125361846",
        "eventCategory": "Management",
        "tlsDetails": {
            "tlsVersion": "TLSv1.3",
            "cipherSuite": "TLS_AES_128_GCM_SHA256",
            "clientProvidedHostHeader": "ec2.us-east-1.amazonaws.com"
        },
        "sessionCredentialFromConsole": "true"
    }
}