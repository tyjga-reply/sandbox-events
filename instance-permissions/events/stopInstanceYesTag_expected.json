{"statusCode": 200, "body": "{\"message\": \"Resource ID extracted successfully!\", \"eventName\": \"StopInstances\", \"resourceId\": \"i-093127c3e6bddd5a6\", \"valid\": true, \"ipInfo\": {\"PrivateIpAddress\": \"172.31.88.70\", \"PublicIpAddress\": \"N/A\"}}"}